### DISCUSSION TITLE
mockito-subclass as default instead of mockito-inline in Quarkus 3

### DISCUSSION THREAD

**Discussion by luca-bassoricci** [quarkusio/quarkus#35079] [Q&A] [answered]
I've some tests in Quarkus 2.x that mock/spy final classes and all works fine just adding `mockito-inline` in pom.xml.
After upgrade to Quarkus 3.x (and to Mockito 5.3)  `mockito-inline` is no more available because it was removed and `inline` mock-maker is now the default (before the default was `subclass`).
After upgrade tests fail because in Quarkus 3 the mock-maker is still `subclass` and not `inline`: is using `subclass` a design choice?

Best regards
Luca Basso Ricci

**Comment by alex-martel** [comment, marked as answer]
I think this will answer your question; https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0#fixation-of-the-mockito-subclass-mockmaker

**Reply by luca-bassoricci** [reply]
I have read migration guide multiple times and I have never pay attention of that... :|
and I have reached with the same solution :D
